Index: test/test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test/test.py b/test/test.py
new file mode 100644
--- /dev/null	(date 1666699521907)
+++ b/test/test.py	(date 1666699521907)
@@ -0,0 +1,20 @@
+from misc import load_map
+import numpy as np
+
+
+def test() -> None:
+    """
+    Test if the shape of the given matrix is a valid representation of the environment
+    :return: None
+    """
+    file_name = input("xyz_palya")
+    map = load_map.MapLoader(file_name = file_name)
+    map_data = map.load()
+    print ("The shape of the new matrix:  " + np.shape(map_data))
+    print(map_data)
+
+    return None
+
+
+if __name__ == "__main__":
+    test()
\ No newline at end of file
